GenderPayGap.WebJob/Functions/Functions.SendReminderEmails.cs

```diff
@@ -21,50 +21,51 @@ public void SendReminderEmails([TimerTrigger("0 * * * *")] TimerInfo timer)
         {
             var start = DateTime.Now;
             CustomLogger.Information("SendReminderEmails Function started", start);
+            
+            try
             {
```

Comment:
There was an extra layer of nesting that I think was unnecessary, so I've removed it. I

Reply:
Looks good :-)